cypress/integration/SearchResultsPage/GeneralPageComponents.feature

```diff
@@ -0,0 +1,253 @@
+Feature: Clinical Trials Search Results Page Components
+
+    ### delighters####
+
+    Scenario: When user is on a search results page he can see all its items
+        Given the user navigates to "/about-cancer/treatment/clinical-trials/search/r?loc=0&rl=1"
+        Then the page title is "Clinical Trials Search Results"
+        And trial info displayes "Results 1-10  of 6821 for your search for: \"all trials\""
+        And "Start Over" link has a href "/about-cancer/treatment/clinical-trials/search"
+        And result list is displayed
+        And there are 2 delighters present
+        And the following delighters are displayed
+            | delighter    | href                                                       | title                              | text                                                                  |
+            | cts-livehelp | /contact                                                   | Have a question?We're here to help | Chat with us: LiveHelpCall us: 1-800-4-CANCER(1-800-422-6237)         |
+            | cts-which    | /about-cancer/treatment/clinical-trials/search/trial-guide | Which trials are right for you?    | Use the checklist in our guide to gather the information you’ll need. |
+
+    # #### Results list manipulation
+
+    Scenario: user can get details about each trial returned by a search and select trials
```

Comment:
The checkbox checking is really for the print feature. So this can be pulled from the commit.
Otherwise, you are not testing that the check boxes maintain their check state between moving between pages.

Reply:
Pulled that scenario out along with steps implementation